lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -139,16 +140,22 @@ export const migrateFileRecord = async (
  * @param {GranulesAndFilesMigrationSummary} granuleAndFileMigrationSummary
  * @param {Knex} knex
  * @param {number} loggingInterval
+ * @param {string} stackName
+ * @param {string} bucket
  * @returns {Promise<MigrationSummary>} - Migration summary for granules and files
  */
 export const migrateGranuleAndFilesViaTransaction = async (
   dynamoRecord: AWS.DynamoDB.DocumentClient.AttributeMap,
   granuleAndFileMigrationSummary: GranulesAndFilesMigrationSummary,
   knex: Knex,
-  loggingInterval: number
+  loggingInterval: number,
+  stackName?: string,
+  bucket?: string
```

Comment:
I recognize that this function's parameter list is getting longer and I am wondering if I've now added too many parameters to it. Suggestions or comments welcome

Reply:
Generally when they get > 3 or so arguments, it's worth considering making it a params object.   In this case, *particularly* as you have stackName/Bucket as optional values.

Reply:
So something like 

```
const someFunc = async ( params: { 
  foo: string, 
  bar: string 
}) => { 
  const { foo, bar } = params;
......
```